doc/how_to/interactivity/bind_component.md

```diff
@@ -0,0 +1,53 @@
+# Binding on components
+
+This guide addresses how to bind parameters and bound functions on components.
```

Comment:
@droumis asking you as I think you may know the answer to this question: do we have a special way to format the word `parameter` when it refers to a Param Parameter?

Reply:
in general, we should probably settle on something like [here](https://pyviz-dev.github.io/panel/how_to/param/uis.html), but I'm not sure we want that in this sentence as these parameters being Param Parameters is not something the user should care about in this particular context, right? It may seem like a bit of a distraction by over-specification. I don't care that much either way

Reply:
It seems to me we have to be pretty explicit at least in the explanations when Parameters are involved. The first example binds `slider.param.value` to  `tabulator.param.page_size`. Because Panel is smart you don't need to specify `slider.param.value`, `slider` is enough.

Reply:
Thanks for bringing this up. We really need an explanation page that distills the Param docs down to what Panel users should care about.

Reply:
[explanation>param](https://pyviz-dev.github.io/panel/explanation/dependencies/param.html) is a good place for it!